### ISSUE TITLE
Check for Admin to delete links and clear pool

### ISSUE THREAD

**Issue by clemsos** [QiHardware/scale-telegram-bot#5]
Only admins should be able to delete links

**Comment by rejon** [comment]
Yeah need this for prepping release...are there admin commands? How can tell who is admin...for now you can hard code me and you.
